libbeat/outputs/mode/modeutil/modeutil.go

```diff
@@ -0,0 +1,127 @@
+package modeutil
```

Comment:
Looks like these changes were copied over  from mode.go, so will not have a closer look.